pyproject.toml

```
@@ -0,0 +1,72 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+version = "1.1.5"
+name = "asyncua"
+description = "Pure Python OPC-UA client and server library"
+requires-python = ">=3.10"
+readme = "README.md"
+license = { text = "GNU Lesser General Public License v3 or later" }
+authors = [
+    { name = "Olivier Roulet-Dubonnet", email = "olivier.roulet@gmail.com" },
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 or later (LGPLv3+)",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+dependencies = [
+    "aiofiles",
+    "aiosqlite",
+    "cryptography>42.0.0",
+    "pyOpenSSL>23.2.0",
+    "python-dateutil",
+    "pytz",
+    "sortedcontainers",
+    "typing-extensions",
+    "wait_for2==0.3.2",
```

maybe
```suggestion
    "wait_for2==0.3.2; python_version < "3.12",
```
since for Python 3.12+ it will never be imported https://github.com/FreeOpcUa/opcua-asyncio/blob/add6eddaab8ed90ffbfaf128c2456bda302ed86e/asyncua/common/utils.py#L153

cf https://packaging.python.org/en/latest/specifications/dependency-specifiers/#dependency-specifiers

---

thanks